cli/cmd/proxy/envoy/http.go

```diff
@@ -1,8 +1,10 @@
-package read
+package envoy
```

Comment:
I think this makes more sense being placed in `cli/common/`. The pattern for the `cmd` directory so far has been to have each directory correspond to a command directly.

For instance `cmd/proxy/read/` corresponds to the CLI command `consul-k8s proxy read`. Having `cmd/proxy/envoy/` implies to me that `consul-k8s proxy envoy` is a command and its code is contained here.

I like that you are taking the opportunity to refactor these things out while you are adding new functionality, but I think it makes sense to store that in `common/` where "shared functionality" has been placed.

Reply:
that makes sense, I kept it here initially because it's really only shared between the proxy commands but let me move this into the `common` package

Reply:
Now that I'm looking at this more, because it's not common across multiple commands, I think it works better as a separate file in the `cli/cmd/logLevel/` directory.

Reply:
so we use this here and in the `read` command

Reply:
so I ended up moving this into the `common` package since it's used in both the `read` and `log` commands